db/create_tables.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS `audit` (
+CREATE TABLE IF NOT EXISTS `audit_records` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `state_machine_name` VARCHAR(255) NOT NULL,
```

Comment:
Thinking to use UUID to identify state machine instance, that's why kept it as varchar.
Have put state machine name to maintain mapping between state machine and state machine instance id for offline query purposes, will remove it from here as it is available in checkpoint table.